cmake/Findmlpack.cmake

```diff
@@ -0,0 +1,18 @@
+IF(NOT DEFINED mlpack_DIR)

```

Comment:
mlpack should come with a cmake config file, so this findmlpack should not be needed
https://github.com/mlpack/mlpack/blob/master/CMake/mlpack-config.cmake.in